packages/wallet/ui/src/util/keyManagement.js

```diff
@@ -391,6 +412,43 @@ export const makeInteractiveSigner = async (
       return tx;
     },
 
+    /**
+     * Sign and broadcast Provision for a new smart wallet
+     *
+     * @throws if account does not exist on chain, user cancels,
+     *         RPC connection fails, RPC service fails to broadcast (
+     *         for example, if signature verification fails)
+     */
+    submitProvision: async () => {
+      const { accountNumber, sequence } = await signingClient.getSequence(
+        address,
+      );
+      console.log({ accountNumber, sequence });
+
+      const b64address = toBase64(toAccAddress(address));
+
+      const act1 = {
+        typeUrl: SwingsetMsgs.MsgProvision.typeUrl,
+        /** @type {Provision} */
+        value: {
+          address: b64address,
+          nickname: 'my wallet',
+          // XXX dry with PowerFlags const
```

Comment:
how about now?

Reply:
One concern I have is when we import these packages just to use a const or type etc, we increase the bundle size by a lot. Maybe it's better to just leave it?

Reply:
Looks like it does have tree shaking yes. Importing this only increased the bundle size by ~4kB. Still the bundle size is getting up there, so I created a separate issue for that https://github.com/Agoric/agoric-sdk/issues/6291